iree/samples/simple_embedding/static_embedding_module.h

```diff
@@ -0,0 +1,19 @@
+// Auto-generated static library header file.
+
+#ifndef IREE_GENERATED_STATIC_EXECUTABLE_LIBRARY_SIMPLE_MUL_DISPATCH_0_
+#define IREE_GENERATED_STATIC_EXECUTABLE_LIBRARY_SIMPLE_MUL_DISPATCH_0_
+#include "iree/hal/local/executable_library.h"
```

Comment:
when generating, add an extra newline between the header guard and the first include